client-bagel-buddy/src/containers/teacher/lesson/canceled/canceled.js

```diff
@@ -1,68 +1,57 @@
-            id: 1,
-            name: "Devon Lane",
-            nationality: "English",
-            timeSlot: "Sep 20, Wed, 20:00 - 20:30",
-            content: "Hi, I want to practice speaking about topics like introducing Japanese culture, travel, and current events. I'm also happy to talk about anything else that you're interested in.",
-            meetLink: "https://meet.google.com/zzw-cqtw-nkt",
-            imgUrl: user_img_url + "6.png"
-        }, {
-            id: 2,
-            name: "Arlene McCoy",
-            nationality: "Portugese",
-            timeSlot: "Sep 20, Wed, 20:00 - 20:30",
-            content: "Hi, I want to practice speaking about topics like introducing Japanese culture, travel, and current events. I'm also happy to talk about anything else that you're interested in.",
-            meetLink: "https://meet.google.com/zzw-cqtw-nkt",
-            imgUrl: user_img_url + "5.png"
-        }, {
-            id: 3,
-            name: "Cody Fisher",
-            nationality: "Portugese",
-            timeSlot: "Sep 20, Wed, 20:00 - 20:30",
-            content: "Hi, I want to practice speaking about topics like introducing Japanese culture, travel, and current events. I'm also happy to talk about anything else that you're interested in.",
-            meetLink: "https://meet.google.com/zzw-cqtw-nkt",
-            imgUrl: user_img_url + "8.png"
-        }])
+        getCancelledLesson()
+            .then((res) => {
+                if (res.success) {
+                    let data = res.data.map((lesson) => {
+                        lesson.lesson.timeslotStart = new Date(lesson.lesson.timeslotStart)
+                            .toLocaleString("en-US", { timeZone: "America/Chicago" });
+                        return lesson;
+                    });
+                    setLessonList(data);
+                } else {
+                    alert(res.message);
+                }
+            });
     }, []);
 
     return (
         <>
             {
-                lessonList.map((lesson) => (
-                    <div className='lesson-row' key={lesson.id}>
+                lessonList.map((lessonObject) => (
+                    <div className='lesson-row' key={lessonObject.lesson._id}>
                         <div className='profile-img'
-                            style={{ backgroundImage: `url(${lesson.imgUrl})` }} />
+                            style={{ backgroundImage: `url('../../images/${lessonObject.student.img_url}')` }} />
                         <div className='lesson-content'>
                             <div className='content-title'>
                                 <div className='title-text'>
-                                    <h2>{lesson.name}</h2>
-                                    <p>{lesson.nationality}</p>
+                                    <h2>{lessonObject.cancel.canceler.name}</h2>
+                                    <p>{lessonObject.cancel.canceler.native_language}</p>
```

Comment:
I remember we are putting student here regardless of who the canceller is. Canceller only showed in the pop-up 
Better double check with Yuli.